cloud-gateway-service/src/test/java/org/zowe/apiml/cloudgatewayservice/acceptance/xForwardHeaders/XForwardedHeadersProxyTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.cloudgatewayservice.acceptance.xForwardHeaders;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.TestPropertySource;
+import org.zowe.apiml.cloudgatewayservice.acceptance.common.AcceptanceTest;
+import org.zowe.apiml.cloudgatewayservice.acceptance.common.AcceptanceTestWithMockServices;
+import org.zowe.apiml.cloudgatewayservice.acceptance.common.MockService;
+import org.zowe.apiml.cloudgatewayservice.filters.X509awareXForwardedHeadersFilter;
+
+import java.io.IOException;
+
+import static io.restassured.RestAssured.given;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+@AcceptanceTest
+@TestPropertySource(properties = {
+    "apiml.security.forwardHeader.trusted-proxies="
+})
+class XForwardedHeadersProxyTest extends AcceptanceTestWithMockServices {
```

Comment:
Please format the indent by meaning and change `Then` to `then` in the test name, ie. `whenNoXForwardHeadersInRequest_ThenXForwardHeadersCreated`